Vagrantfile

```diff
@@ -2,7 +2,8 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "chef/centos-6.5"
+  config.vm.box = "chef/centos-6.6"
+  #config.vm.box = "bhosmer/centos6.6-minimal"
```

Comment:
We can keep the GUI off. I only turn the GUI on if i can't get something
working and I need a UI.
On Jun 16, 2015 5:54 PM, "Kit Plummer" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/ngageoint/geowave-vagrant/pull/3#discussion_r32575744>
> :
>
> > @@ -2,7 +2,8 @@
> >  # vi: set ft=ruby :
> >
> >  Vagrant.configure("2") do |config|
> > -  config.vm.box = "chef/centos-6.5"
> > +  config.vm.box = "chef/centos-6.6"
> > +  #config.vm.box = "bhosmer/centos6.6-minimal"
>
> Yeh, it shouldn't be there. Will fix this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ngageoint/geowave-vagrant/pull/3/files#r32575744>.
>